inf-terraform-aws/ocp.env

```diff
@@ -0,0 +1,4 @@
+MEMORY_LIMIT=128Mi
+MEMORY_REQUEST=128Mi
+CPU_REQUEST=50m
+CPU_LIMIT=100m
```

Comment:
I think this file is not needed. It's used in https://github.com/opendevstack/ods-quickstarters/blob/2f954388379b770419e00e20b2e81521ddafa0ec/common/scripts/create-component.sh#L128, which is only used to create OpenShift components - but you don't have any for this quickstarter, right?

Reply:
@nichtraunzer can you look into this comment?